Lib/test/test_ast.py

```diff
@@ -583,6 +583,19 @@ def test_literal_eval_complex(self):
         self.assertRaises(ValueError, ast.literal_eval, '3+(0+6j)')
         self.assertRaises(ValueError, ast.literal_eval, '-(3+6j)')
 
+    def test_literal_eval_message(self):
+        # Issue #32888
+        tests = {
+            "2 * 5": "BinOp not allowed in literal",
+            "[] + []": "List not allowed in binary +/-",
+            "+''": "Str not allowed in unary +/-",
+            ast.BinOp(ast.Num(1), ast.Add(), 'oops'): "malformed node or string: 'oops'",
+        }
+        for test, message in tests.items():
+            with self.assertRaises(ValueError) as cm:
```

Comment:
A helpful thing to use in between lines 594 and 595 would be `with self.subTest(test=test, message=message):` and then indent the parts below it. That way each of these end up being tested independently rather than potentially having the items loop have a case fail and cause the rest of the cases to fail to execute. 

Check out https://docs.python.org/3/library/unittest.html#distinguishing-test-iterations-using-subtests